### DISCUSSION TITLE
Meeting: December 17th, 2024

### DISCUSSION THREAD

**Discussion by earth2marsh** [OAI/sig-moonwalk#165] [Announcements]
## Weekly meetings happen on Tuesdays

**Time:** 5pm GMT / 9am Pacific
**Zoom:** https://zoom-lfx.platform.linuxfoundation.org/meeting/98869769538?password=d769f940-7da7-4de8-9fab-685a1cc8f33e
[NB: the above meeting was changed to the LFX system on Oct 22, 2024. (remove this note in 2025)]

### Accessibility & Etiquette

* Participants must abide by our [Code-of-Conduct](https://github.com/OAI/sig-moonwalk?tab=coc-ov-file).
* Meetings are recorded for future reference, and for those who are not able to attend in-person.
* We invited challenges to language or behaviour that is harmful or not inclusive during this meeting.
* We look forward to your participation, but please consider these acts of etiquette:
  * Remain on mute when not speaking to prevent interruptions.
  * Blur your background to reduce visual distractions.
  * Use the Zoom meeting "Raise Hand" feature to notify the group when you wish to speak.

| Blur My Background | Raise Hand |
|-|-|
| <img width="323" alt="Screenshot of Zoom UI showing the 'Stop Video' and 'Blur My Background' control" src="https://github.com/OAI/OpenAPI-Specification/assets/7367/7e43dbbb-6529-46e6-8b04-4c1aa852d9dd"> | <img width="323" alt="Screenshot of Zoom UI showing the 'Reaction' and 'Raise Hand' control" src="https://github.com/OAI/OpenAPI-Specification/assets/7367/f991722f-4651-40aa-9bc4-7e9a2a165a6a"> |

### Agenda Structure

| Topic | Owner | Decision/NextStep |
|-------|---------|---------|  
| | | |
Intros and governance meta-topics (5 mins) | SIG members | -- |
General discussion topics (list below) | SIG members | -- |
Wrap-up and seed the next agenda (5 mins) | SIG members | -- |

Whether attending or not, **anyone can comment on this issue prior to the meeting to suggest topics or to add comments on planned topics or proposals**.

**Comment by earth2marsh** [comment]
@daveshanley will return for a deeper dive into The Doctor, Vacuum, and libopenapi

**Comment by darrelmiller** [comment]
Notes from conversations with Dave.
- knowing line number and position of every element is important for many scenarios
- ordering of collections can be significant.
- Maintaining yaml comments
- YAML references?
- [is maintaining of object member order important?]
- Rendering linting errors detected in the "description" back in the originating "document"
- Uses Node instead of making a distinction between 

Could we use the libopenapi model as the basis for DOM API and ADA API?

**Comment by earth2marsh** [comment]
[Video](https://zoom.us/rec/share/cryUifxdpz2oxSeVnE_V_LW0Y7Kw_MU067NAgxrE66tJuHu8P455jrw0CBZgTQSO.DTjcQS5bYyUBGIav) is available.
